im.srain.Srain.json

```diff
@@ -0,0 +1,146 @@
+{
+  "app-id": "im.srain.Srain",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "srain",
+  "rename-desktop-file": "Srain.desktop",
+  "rename-appdata-file": "Srain.appdata.xml",
+  "rename-icon": "srain",
+  "copy-icon": true,
+  "finish-args": [
+    "--share=ipc",
+    "--device=all",
```

Comment:
what devices does this need access to? Looking at the website it shouldn't need more than dri permissions.

Reply:
I mistakenly believed this was necessary for keyboard input, I'll remove it.